app/src/main/java/com/tohami/newsapi/adapter/BindingAdapters.java

```diff
@@ -0,0 +1,19 @@
+package com.tohami.newsapi.adapter;
+import androidx.databinding.BindingAdapter;
+
+import android.graphics.drawable.Drawable;
+import android.widget.ImageView;
+
+import com.squareup.picasso.Picasso;
+
+public class BindingAdapters {
```

Comment:
Remove data binding from the project and update it accordingly

Reply:
done